lib/options.nix

```diff
@@ -72,7 +72,7 @@ rec {
           ''
       );
 
-    mkNullableWithRaw = type: mkNullable (maybeRaw type);
+    mkNullableWithRaw = type: mkNullable (nixvimTypes.maybeRaw type);
```

Comment:
Not sure that this is need.

Reply:
A that is a leftover. but it will error out without it. But `mkNullableWithRaw` is never used anywhere so it doesn't normally turn up.

Reply:
```
 error: undefined variable 'maybeRaw'
        at /nix/store/p39jzjnq4qa2v2vnfdwlzv05zmn59083-source/lib/options.nix:75:43:
            74|
            75|     mkNullableWithRaw = type: mkNullable (maybeRaw type);
              |                                           ^
            76|
```

Reply:
Nice catch indeed. Fixed in #1526

Reply:
Merged ! Can you rebase on `main` please ?

Reply:
done